### DISCUSSION TITLE
Weekly Development Meeting

### DISCUSSION THREAD

**Discussion by waltsims** [waltsims/k-wave-python#23] [Announcements]
Hello k-Wave-python developers,

We are meeting weekly to discuss development topics around k-wave-python 🚀.

Anyone interested in getting involved with the project can attend in the following [Zoom room Thursdays at 7:30 a.m. PST](https://us05web.zoom.us/j/86108761081?pwd=O5ztApJda5CokCzzTn5Sl29OwfpZfO.1)

**Comment by waltsims** [comment]
The update meeting for tomorrow the 4th of November is canceled :)

**Reply by ghafoorharis** [reply]
Hey @waltsims !
do you still organize these meetings? I would love to join it.
Thanks

**Reply by waltsims** [reply]
> Hey @waltsims !
> 
> do you still organize these meetings? I would love to join it.
> 
> Thanks

Hi @ghafoorharis 
Yes our meetings are currently scheduled for every Thursday 6 am PST. Here is the current link:

http://tiny.cc/v7sf001

**Reply by waltsims** [reply]
No meeting tomorrow. @faridyagubbayli and I have conflicts. @ghafoorharis I hope to see you next week.

**Reply by waltsims** [reply]
No meeting today. Personal reasons.
